docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md

```diff
@@ -9,7 +9,7 @@ Please refer to [02-Integrate to Rancher from Harvester settings](https://harves
 3. Click import existing
 4. Copy the curl command 
 ![image.png](https://images.zenhubusercontent.com/61519853321ea20d65443929/08e70d37-e573-47b1-a3d6-0f3615116d48)
-1. SSH to harvester master node (user: rancher)
+1. SSH to harvester master node
 1. Execute the curl command to import harvester to rancher
 `curl --insecure -sfL https://192.168.50.82/v3/import/{identifier}.yaml | kubectl apply -f -`
```

Comment:
This isn't how we import Harvester clusters anymore. It should be
1. Create a new Harvester cluster from Virtualization Management
1. Change the `cluster-registration-url` to match the one in Rancher

Reply:
Thanks @noahgildersleeve @albinsun I removed the file.